src/styles/index.scss

```diff
@@ -1,3 +1,143 @@
-body {
+  @import'/src/styles/stars';
+
+* {
+   box-sizing: border-box;
+ }
+
+ body,
+ h2,
+ p,
+ div{
   margin: 0;
-}
+ }
+
+ .card {
+   width: 200px;
+  //  height: 406px;
+   flex-shrink: 0;
+   border-radius: 5px;
+   border: 1px solid #f3f3f3;
+   background: #fff;
+   position: relative;
+
+   &__image {
+     margin-top: 32px;
+     margin-left: 19px;
+     width: 160px;
+     height: 134px;
+     background-image: url(../images/imac.jpeg);
+     background-size: cover;
+     background-position: center;
+   }
+
+   &__header,
+   &__product-code {
+     margin-left: 16px;
+   }
+
+   &__header {
+    font-family: Roboto, sans-serif;
+     margin-top: 36px;
+     margin-bottom: 4px;
+     width: 166px;
+     color: var(--main-accent, #060b35);
+     font-style: normal;
+     font-size: 12px;
```

Comment:
![2024-01-03_15h22_20](https://github.com/mate-academy/layout_product-cards/assets/61904995/6faff7e3-5737-4d7a-bbe1-7b17dd42fb9c)

Reply:
isn't fixed. you shouldn't redeclare default browser values